README.md

```diff
@@ -1,24 +1,218 @@
-# README
+<a name="readme-top"></a>
```

Comment:
# Frontend review

- [ ] This command won't work:

![image](https://github.com/unorjikingsley/Reservation-API/assets/57421823/289e6d8c-eef0-41e2-879b-e67e5c40e5ec)

The correct command is `npm run dev`

- [ ]  Please, kindly refactor the "App" and "DeleteCar" functions in your project to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

![image](https://github.com/unorjikingsley/Reservation-API/assets/57421823/f1f5fb80-6ac6-4729-96d5-f5675cd381ff)

![image](https://github.com/unorjikingsley/Reservation-API/assets/57421823/6f75a4ec-a0c2-4a24-80af-b6a204cc6ac6)


```javascript
const App = () => {
  // ...
}
```


```javascript
const DeleteCar= () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

That way, your code will meet the latest standards. 💪